src/routes/Receive/ReceiveMoney.tsx

```diff
@@ -128,6 +128,7 @@ const ReceiveMoney: FC = () => {
             accountId={account?.id || state?.accountId}
             onSelectOption={setAccount}
             mb="2rem"
+            includeViewOnly={true}
```

Comment:
nit (non-blocking): Just fyi, this could just be `includeViewOnly` since an attribute name with no value defaults to `true`.

Reply:
Cool did not know that, ty!